app/marketing/management/commands/expiration_start_work.py

```diff
@@ -78,6 +79,7 @@ def handle(self, *args, **options):
                         if not actions:
                             should_warn_user = True
                             should_delete_interest = False
+                            last_heard_from_user_days = (datetime.now() - interest.created.replace(tzinfo=None)).days
```

Comment:
Can we use `tzinfo=pytz.UTC` here?

Reply:
yes we can make that change.. but the problem is that the rest of the datetime's in the `expiration_start_work` flow are not timezone aware..   let me see if it's an easy lift to do this everywhere

Reply:
965505b67c1b6c8035e276db61b814375ed14cef